src/main/webapp/room.js

```diff
@@ -0,0 +1,19 @@
+let players = [];
+
+const createUser = (ev)=>{
```

Comment:
does this createUser variable need to be created? I see you are passing it in to line 18. Does it still work if you just had it as a simple function?

```
function createUser(ev) {
}
```

It's been a while since I've done any JS so not sure what the benefit is of the const

Reply:
@alekseychaikovsky Yes it still works as a simple function. People use arrow functions a lot online and in videos so I just use them too, and I've heard it's better to use const over let. I can make it regular function that's better?